modules/nvidia_plugin/tests/unit/cuda_multi_graph_ti_test.cpp

```diff
@@ -4,12 +4,21 @@
 
 #include <gtest/gtest.h>
 
+#include "common_test_utils/node_builders/eltwise.hpp"
+#include "common_test_utils/node_builders/gru_cell.hpp"
+#include "common_test_utils/test_constants.hpp"
 #include "cuda_graph_topology_runner.hpp"
 #include "cuda_simple_execution_delegator.hpp"
-#include "ops/parameter.hpp"
-#include "ops/result.hpp"
-#include "ov_models/builders.hpp"
+#include "functional_test_utils/ov_plugin_cache.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/parameter.hpp"
+#include "openvino/op/result.hpp"
+#include "openvino/op/split.hpp"
+#include "openvino/op/squeeze.hpp"
+#include "openvino/op/unsqueeze.hpp"
 #include "ov_models/utils/data_utils.hpp"
+#include "template/properties.hpp"
```

Comment:
I wonder how compilation has passed with this line.. This file is not available in the repo.